src/content/structured/community/roadmap.mdx

```diff
@@ -22,57 +22,52 @@ See our [GitHub](https://github.com/orgs/mi6/projects/2/views/1) for more detail
 
 ## What's new
 
-We recently added:
+New features for the UI Kit:
 
-- [Toasts](/components/toasts).
-- [Cards V2](/components/cards), with images and expandable area.
-- Asynchronous filtering for [search bar](/components/search-bar).
-- [Compact stepper](/components/stepper).
-- [Chips](/components/chips).
-- [Popover menu](/components/popover-menu).
+- Released the [Toast](/components/toasts) component.
+- Released the [Cards V2](/components/cards) component, with images and expandable area.
+- Released the [Compact stepper](/components/stepper) component.
+- Released the [Chips](/components/chips) component.
+- Released the [Popover menu](/components/popover-menu) component.
+- Asynchronous filtering for [search bar](/components/search-bar) component.
 
 ## Working on now
 
-We're working to:
```

Comment:
I think we should keep this line instead of the new line as it matches the tone of the other paragraphs better